src/Task/ComposerNormalize.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Task\AbstractExternalTask;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Class ComposerNormalize.
+ */
+class ComposerNormalize extends AbstractExternalTask
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function canRunInContext(ContextInterface $context)
+    {
+        return $context instanceof GitPreCommitContext || $context instanceof RunContext;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getConfigurableOptions()
+    {
+        $resolver = new OptionsResolver();
+        $resolver->setDefaults([
+            'indent_size' => 4,
+            'indent_style' => 'space',
+            'update_lock' => true,
+            'verbose' => false,
+        ]);
+
+        $resolver->addAllowedValues('indent_style', ['tab', 'space']);
+        $resolver->addAllowedTypes('indent_size', ['int']);
```

Comment:
Note that when `indent_style` is specified, `indent_size` needs to be specified as well (and vice versa).

Not sure if this should be handled here already.

Reply:
I guess we can leave it like this for now.

Reply:
The default value will always fill those properties anyway.